package.json

```diff
@@ -76,6 +73,8 @@
     "webpack-stream": "^3.2.0"
   },
   "dependencies": {
+    "@types/react": ">=15 <16",
+    "moment": "2.16.x",
```

Comment:
We're aware `moment` is not a dependency of this project, and we would like it to stay that way :)